specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2021-01-01/vaults.json

```diff
@@ -1375,8 +1375,8 @@
       "type": "object",
       "description": "A resource identity that is managed by the user of the service.",
       "properties": {
-        "principalId": {
-          "description": "The principal ID of the user-assigned identity.",
+        "objectId": {
```

Comment:
>objectId [](http://example.com/codeflow?start=9&length=8)

Same comment as the with the previous version, according to our docs, principalId is the appropriate name here.

Reply:
Yes i will be changing it asap and reverting it back to principal id

Reply:
Updated